### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Mosuswalks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+A Flippy Flop is an unpredictable data structure.
+Flip (insertion) and flop (removal) operations succeed at random.
+
+## Usage
+
+As you'll see in the example below, the Flippy Flop is sufficiently unpredictable.
+
+### Scala
+
+Scala Code Here?
```
Comment (shakeelmohamed) [inline README.md]: Thanks @Mosuswalks, any chance you want to take a crack at adding the scala example?

**Review by shakeelmohamed — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Mosuswalks) [reply README.md]: Hey @shakeelmohamed, I would love to give it a try!
I'm not familiar with Scala but it seems to be somewhat similar to Java.

Do you have the Scala Class i can reference?

Comment (shakeelmohamed) [reply README.md]: @Mosuswalks This is the only program I've ever written in Scala so you're not far behind 😄 
The code is on the develop branch: https://github.com/flippy-flop/ff-scala/blob/develop/FlippyFlop.scala

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+var flop1 = ff.flop
+
+// Could be 5 if the flop call didn't remove the value, or undefined if the flip call failed
+var flop2 = ff.flop
+
+// However, it's easy to verify the number of elements in a FlippyFlop
+var count = ff.length
+
+```
+# Contact
```
Comment (shakeelmohamed) [inline README.md]: we'll need a blank line after the backticks

Comment (Mosuswalks) [reply README.md]: On it!

**Comment by shakeelmohamed** [comment]
No idea if this actually works, but I'll take it. Thanks a ton @Mosuswalks!

**Pull request merged**